libres/lib/include/ert/enkf/enkf_defaults.hpp

```diff
@@ -26,16 +26,10 @@
 /**
   Default file name for export file for GEN_KW parameters
 */
-#ifdef __cplusplus
-extern "C" {
-#endif
 #define DEFAULT_GEN_KW_EXPORT_NAME "parameters"
 
 const char *enkf_defaults_get_default_gen_kw_export_name();
```

Comment:
Kill this func. The default is a constant. Why call a function when you can just return the string.

Reply:
Thought there must be a better solution - but am apparently bad at googling - will do!